core/ui/src/views/ApplicationsCenter.vue

```diff
@@ -0,0 +1,1051 @@
+              @updatePage="tablePage = $event"
+            >
+              <template slot="empty-state">
+                <template v-if="hasActiveFilters && modules.length">
+                  <!-- no search results -->
+                  <NsEmptyState
+                    :title="$t('common.no_search_results')"
+                    key="no-results-empty-state"
+                  >
+                    <template #description>
+                      <div class="flex flex-col items-center text-center gap-2">
+                        <p>
+                          {{ $t("common.no_search_results_description") }}
+                        </p>
+                        <NsButton
+                          kind="ghost"
+                          size="field"
+                          @click="clearFilters()"
+                          >{{ $t("common.clear_filters") }}
+                        </NsButton>
+                      </div>
+                    </template>
+                  </NsEmptyState>
+                </template>
+                <template v-else>
+                  <!-- no applications installed -->
+                  <NsEmptyState :title="$t('applications.no_application')">
+                    <template #description>
+                      <div>
+                        {{ $t("applications.no_application_description") }}
+                      </div>
+                      <NsButton
+                        kind="primary"
+                        class="empty-state-button"
+                        :icon="ArrowRight20"
+                        @click="goToSoftwareCenter"
+                        :disabled="loading.listModules"
+                        >{{ $t("applications.go_to_software_center") }}
+                      </NsButton>
+                    </template>
+                  </NsEmptyState>
+                </template>
+              </template>
+              <template slot="data">
+                <cv-data-table-row
+                  v-for="(row, rowIndex) in tablePage"
+                  :key="`${rowIndex}`"
+                  :value="`${rowIndex}`"
+                >
+                  <cv-data-table-cell>
+                    <span>
+                      {{
+                        row.ui_name ? row.ui_name + " (" + row.id + ")" : row.id
+                      }}
+                    </span>
+                    <cv-interactive-tooltip
+                      v-if="row.ui_note"
+                      alignment="start"
+                      direction="right"
+                      class="info margin-left-sm"
+                    >
+                      <template slot="content">
+                        <span class="underline-tooltip">
+                          {{ row.ui_note }}
+                        </span>
+                      </template>
+                    </cv-interactive-tooltip>
+                  </cv-data-table-cell>
+                  <cv-data-table-cell>
+                    <img
```

Comment:
The background of the logo switches to white on mouse hover. Furthermore `module-logo` CSS class seems to have too many CSS rules 🤔